src/main/java/net/smartcosmos/dao/objects/converter/ObjectResponseToObjectEntityConverter.java

```diff
@@ -0,0 +1,42 @@
+package net.smartcosmos.dao.objects.converter;
+
+import net.smartcosmos.dao.objects.domain.ObjectEntity;
+import net.smartcosmos.dto.objects.ObjectResponse;
+import net.smartcosmos.security.user.SmartCosmosUser;
+import net.smartcosmos.security.user.SmartCosmosUserHolder;
+
+import net.smartcosmos.util.UuidUtil;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.format.FormatterRegistrar;
+import org.springframework.format.FormatterRegistry;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author voor
```

Comment:
I dislike the @author annotation also - perhaps I have missed to remove it while duplicating a class for modification. In fact, we do not need @author, because we can use Git to track all changes.
In IntelliJ I have removed the author entry from the new file template (but this doesn't help when duplicating classes).